GVFS/GVFS.FunctionalTests/Program.cs

```diff
@@ -122,6 +122,17 @@ private static void RunBeforeAnyTests()
                 {
                     File.WriteAllText(statusCacheVersionTokenPath, string.Empty);
                 }
+
+                string upgradeDownloadPath = Path.Combine(
+                    Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData),
+                    "GVFS",
+                    "GVFS.Upgrade",
+                    "Downloads");
+
+                if (Directory.Exists(upgradeDownloadPath))
+                {
+                    Directory.Delete(upgradeDownloadPath, recursive: true);

```

Comment:
This can throw exceptions, and its recursive implementation is quite unreliable. We have our own method for doing a reliable recursive directory delete.

Reply:
This not needed anymore, removed. Updated Functional Tests to ignore reminder messaging.